src/pages/hegel/reference/mechanical-object-section-1.mdx

```diff
@@ -0,0 +1,18 @@
+
+In a somewhat throwaway remark at the end of this conceptual discussion, Hegel writes that the “mechanical object” “does not consist of *atoms*, for these are not objects because they are not totalities” (712).  It is worth musing on this brief remark because it reveals a lot about how Hegel conceptualises the wider significance of his account of the “mechanical object”. Hegel clearly has some mechanistic, in particular atomistic, worldviews in his crosshairs when he goes out of his way to state that the “mechanical object” does not consist of atoms. Hegel’s target is those that theorise that the world is fundamentally made out of atoms and that since the world is made out of atoms that everything is really just atoms.  In Hegel’s account of “Mechanism”, he goes out of his way to explicitly state that the “mechanical object”, the simplest kind of mechanical entity, is not reducible to atoms. There is not something more fundamental to the “mechanical object” that is the true reality of things. Moreover, and perhaps a stronger rebuke to the atomists, not only is the “mechanical object” not reducible to atoms, but it is also not the most fundamental thing. Hegel rejects this notion of fundamentality out of hand and it is this rejection that is his strongest argument against atomistic or atomistic-like conceptions of reality.
+
+What follows from the last quotation is a long remark about how Hegel’s account of the “mechanical object” is akin to Leibniz’s conception of a monad. Hegel writes:
+
+>“The Leibnizian *monad* would be more of an object since it is a total representation of the world, but confined within its *intensive subjectivity* it is supposed at least to be essentially *one* within itself. Nevertheless, the monad determined as an *exclusive one* is only a principle *that reflection assumes*. Yet the monad is an object, partly in that the ground of its manifold representations-of the developed, that is, the *posited* determinations of its merely *implicit* totality-lies *outside it*, and partly also in that it is indifferent to the monad that it constitutes an object *along with others*; it is thus in fact not *exclusive* or *determined for itself*” (712).
+
+**Here, I invite a scholar with knowledge of Leibniz and his monads to expound on the above paragraph.**
```

Comment:
Do you want to explicitly leave this like that? I think putting a generic "this section is a stub, you can help us improving it by contributing to it" kind of thing, like wikipedia does. What do you think?

Reply:
I think it depends on the tone of the article and, perhaps, the website overall. What I wrote was meant to create a tone of community and also to add an individual touch by specifying what I think needs to be added here. What you're suggesting is more down-the-line and impersonal. 
I like what I've done, but if you think it opens the door to too much interpretation, happy to alter it.

Reply:
The issue is that it appears like a part of the article itself, which it shouldn't be. Moreover, putting it in bold is too striking. I agree that we could do something more personal, so we can use this component (see image below) as a template and then whoever can just add their own bit of text.

![image](https://github.com/systemphil/sphil/assets/118931755/5b31346a-0f27-4d17-b98e-51147a1576ac)